kuma/static/js/utils/post-message-handler.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Beacons events of type 'Performance Events' to GA
+ * @param {Object} data - The data associated with this event
+ */
+function handlePerformanceEvents(data) {
+    // one of hitType: event
+    mdn.analytics.trackEvent({
+        category: data.category,
+        action: data.action,
+        label: new Date().getTime() + '-' + mdn.utils.randomString(5),
+        value: data.value - performance.timing.fetchStart
+    });
+
+    // one of hitType: timing
+    mdn.analytics.trackTiming({
+        category: data.category,
+        timingVar: data.action,
+        value: data.value - performance.timing.fetchStart
+    });
+}
+
+/**
+ * Processes marks and sends beacons to GA based on mark value
+ * @param {Object} perfData - Object containing performance mark/measure information
+ */
+function handlePerfMarks(perfData) {
+    // if there is no `measureName` property, just set a mark
+    if (!perfData.measureName) {
+        window.mdn.perf.setMark(perfData.markName);
```

Comment:
Based on current logic, we'll never run this branch - it is only called when ``measureName`` is defined.

Reply:
Doh!, that should be `markName`

Reply:
👍 I see ``interactiveExamplesMsgHandler`` changed to check for ``markName``.